src/cpu/aarch64/acl_post_ops.hpp

```diff
@@ -32,7 +32,9 @@ struct acl_post_ops_t {
     // init the acl_post_ops_t. Note that this function modifies the passed in
     // post ops by setting the preferred memory formats
     status_t init(engine_t *engine, post_ops_t &post_ops,
-            const memory_desc_t &dst_md) {
+            const memory_desc_t &dst_md, int post_op_start_index_ = 0) {
+
+        post_op_start_index = post_op_start_index_;
```

Comment:
The best practice is to do the opposite - class members are named with underscore and arguments are not.

Reply:
I'm fine to hold off this comment, feel free to decide if you want to follow that.

Reply:
Good point, I've fixed this, and try to enforce this in the future.